germany_fidorbank/src/ofxstatement/plugins/germany_fidorbank.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import hashlib
+import re
+
+from ofxstatement.plugin import Plugin
+from ofxstatement.parser import CsvStatementParser
+from ofxstatement.statement import StatementLine
+
+
+class FidorBankAGPlugin(Plugin):
+    def get_parser(self, filename):
+        encoding = self.settings.get('charset', 'utf-8')
+        with open(filename, 'r', encoding=encoding) as f:
+            lines = f.readlines()
+        parser = FidorBankAGParser(lines)
+        parser.statement.account_id = self.settings['account']
+        parser.statement.bank_id = self.settings.get('bank', '50050201')
```

Comment:
This default `50050201` does not make sense with Fidor Bank, it's the BLZ from 1822direkt

Reply:
For Fidor, looks like it is `70022200`